test/base/ProfilerTest.ooc

```diff
@@ -0,0 +1,33 @@
+use ooc-base
+use ooc-unit
+import io/File
+
+ProfilerTest: class extends Fixture {
+	init: func {
+		super("Profiler")
+		this add("log", func {
+			outputFile := "test/test.log"
```

Comment:
OK, but this file is removed in this test. Should we keep it for manual inspection ?